src/main/java/view/ResultView.java

```diff
@@ -0,0 +1,24 @@
+package view;
+
+import core.Car;
+
+import java.util.List;
+
+public class ResultView {
+
+    private ResultView() {};
+
+    public static void printOutputValue(List<Car> carList) {
+        carList.stream().forEach(car -> System.out.println(getStringFromPosition(car)));
+        System.out.println();
+    }
+
+    public static String getStringFromPosition(Car car) {
+        String string = "";
+        for(int i=0 ; i < car.position ; i++) {
+            string = string.concat("-");
+        }
+        return string;
```

Comment:
```suggestion
        return Stream.Iterate(0, () -> "-")
                     .limit(car.position)
                     .collect(Collectors.joining(""));
```

혹은 

```suggestion
        return IntStream.range(0, car.position)
                     .mapToObj(i -> "-")
                     .collect(Collectors.joining(""));
```

이렇게 표현할 수 있습니다!